kyuubi-common/src/main/scala/org/apache/kyuubi/operation/log/OperationLog.scala

```diff
@@ -34,17 +34,13 @@ import org.apache.kyuubi.session.Session
 import org.apache.kyuubi.util.ThriftUtils
 
 object OperationLog extends Logging {
-  final private val OPERATION_LOG: InheritableThreadLocal[OperationLog] = {
-    new InheritableThreadLocal[OperationLog] {
-      override def initialValue(): OperationLog = null
-    }
-  }
+  final private val OPERATION_LOG = new ThreadLocal[OperationLog]
```

Comment:
I suppose it's not necessary to use `InheritableThreadLocal` here, am I missing something?